client/src/components/ManageArtist/Manage.tsx

```diff
@@ -61,7 +61,9 @@ export const Manage: React.FC = () => {
             margin-top: 0 !important;
           }
           @media screen and (max-width: ${bp.medium}px) {
-            padding: .5rem;
+            padding: var(--mi-side-paddings-xsmall);
+            padding-top: .5rem;
+            padding-bottom: .5rem;
```

Comment:
If the intention in the above ones is to have the padding-top and the padding-bottom overwrite the padding, then this would be the way to do it,

but maybe that's a sign that needs its own variable or component